src/Umbraco.Core/Services/SectionService.cs

```diff
@@ -212,7 +212,7 @@ public Section GetByAlias(string appAlias)
         public void MakeNew(string name, string alias, string icon)
         {
             var sections = GetSections();
-            var nextSortOrder = sections != null ? sections.Max(x => x.SortOrder) + 1 : 1;
+            var nextSortOrder = sections != null && sections.Any() ? sections.Max(x => x.SortOrder) + 1 : 1;

```

Comment:
Can `sections` be null? The method `GetSections()` returns an `IEnumerable` and there's no null check against it in other calls.